docs/more/debugging.md

```diff
@@ -4,11 +4,14 @@ title: Debugging
 
 ## Panics
 
-Please use the [`console_error_panic`](https://github.com/rustwasm/console_error_panic_hook) crate for nicer stacktraces with Rust symbols. Note, that it is not compatible with apps built with `cargo-web`.
+We **strongly recommend** the [`console_error_panic`](https://github.com/rustwasm/console_error_panic_hook) 
+which catches `panic!`s and outputs them to the console. Unfortunately this is not compatible with 
+apps built using `cargo-web`.
```

Comment:
I think we should just mention that `yew::start_app()` does all of this for you.

Reply:
👍🏼